app/pubspec.yaml

```diff
@@ -120,6 +120,7 @@ dependencies:
   flutter_local_notifications: ^16.0.0
   email_validator: ^2.1.17
   app_settings: ^5.1.1
+  flutter_animate: ^4.2.0+1
```

Comment:
I don't find this still being used in this version of the PR. Did I miss it or can this go now?

Reply:
Yes, it is not in used as of now. I have used this library to play around with couple of animation effect in Space Tab but as by changing couple of thing flickering effect is reduced at some extend and tabs looks fine without animation.

I have kept animation library in `pusspec.yaml` as it is good library for animation and we can use it on many of the application areas to have better user experience. 

But ya I can remove it as it is not in any use as of now.